sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/BeamSqlSeekableTable.java

```diff
@@ -28,8 +28,18 @@
  */
 @Experimental
 public interface BeamSqlSeekableTable extends Serializable{
+  /**
+   * prepare the instance.
+   */
+  void setup();
```

Comment:
can we follow the naming of `setUp` & `tearDown` which used by junit, e.g. https://github.com/junit-team/junit4/blob/41d44734f41aba0cf6ba5a11ff5d32ffed155027/src/main/java/junit/extensions/TestSetup.java#L33

Reply:
+1 updated